fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -692,6 +693,14 @@ stmt ::=
     {: RESULT = stmt; :}
     | uninstall_plugin_stmt : stmt
     {: RESULT = stmt; :}
+    | create_sql_block_rule_stmt : stmt
+    {: RESULT = stmt; :}
+    | alter_sql_block_rule_stmt : stmt
+    {: RESULT = stmt; :}
+    | drop_sql_block_rule_stmt : stmt
+    {: RESULT = stmt; :}
+    | show_sql_block_rule_stmt : stmt
+    {: RESULT = stmt; :}
```

Comment:
I think you can put these 4 stmts to `create_stmt`, `drop_stmt`, `alter_stmt` and `show_stmt`.

Reply:
ok